site/en/workflows.md

```diff
@@ -629,12 +629,14 @@ In this processing step, text region images are taken as input and their skew is
 
 <!-- BEGIN-EVAL sed -n '0,/^## Notes/ p' ./repo/ocrd-website.wiki/Workflow-Guide-clipping.md|sed '$d' -->
 In this processing step, intrusions of neighbouring non-text (e.g. separator) or text segments (e.g. ascenders/descenders) into
-text regions of a page can be removed. A connected component analysis is run on every text region,
+text regions of a page (or text lines or a text region) can be removed. A connected component analysis is run on every segment,
 as well as its overlapping neighbours. Now for each conflicting binary object,
 a rule based on majority and proper containment determines whether it belongs to the neighbour, and can therefore
 be clipped to the background.
 
-This basic text-nontext segmentation ensures that for each text region there is a clean image without interference from separators and neighbouring texts. (Cleaning via coordinates would be impossible in many common cases.)
+This basic text-nontext segmentation ensures that for each text region there is a clean image without interference from separators and neighbouring texts. (On the region level, cleaning via coordinates would be impossible in many common cases.) On the line level, this can be seen as an alternative to _resegmentation_.
```

Comment:
the comment on line clipping should be moved to clipping on line level (and not show up on region level)

Reply:
Indeed.

Reply:
Sorry, I don't understand what to move where, can you be more specific?

Reply:
IIUC @EEngl52 meant the last sentence from this change in step 10 (region-level clipping) should be moved into a new paragraph in step 12 (line-level resegmentation).

Since I cannot make suggestions out of context here, do you want me to edit this into the wiki?

Reply:
That would be best, thanks!